### ISSUE TITLE
Create tablet layouts

### ISSUE THREAD

**Issue by dektar** [5calls/android#36]
IssueActivity in particular would look bad on a tablet - the buttons would get super wide.

Probably have big margins for all activities on tablets, with more centered content, for starters.

**Comment by dektar** [comment]
Blocked on UX, but later help is welcome here.

**Comment by cdoremus** [comment]
FYI, the IssueActivity looks fine on my Kindle Fire (see attached).
![screenshot_2017-02-23-23-16-50](https://cloud.githubusercontent.com/assets/942174/23290433/a49afe02-fa1e-11e6-81b3-9d37a4fed741.png)

**Comment by dektar** [comment]
@cdoremus That's the MainActivity. The IssueActivity shows one issue, and the buttons will be super wide in tablet layout. It would probably be better to float the buttons into a grid, or just make them have a maximum width.

**Comment by cdoremus** [comment]
@dektar, you're right. My bad. The buttons look better in a grid, as the web app looks on my tablet:
![screenshot_2017-02-24-00-10-39](https://cloud.githubusercontent.com/assets/942174/23291378/18519afc-fa26-11e6-9520-4c810dd81878.png)

**Comment by dektar** [comment]
The RepCallActivity shows all 3 buttons in a row on wide layouts. Doing more is a nice-to-have but not critical.
